rpmbuild/copr-rpmbuild.spec

```diff
@@ -47,7 +47,7 @@ BuildRequires: %{python_pfx}-munch
 BuildRequires: %{python}-requests
 BuildRequires: %{python_pfx}-jinja2
 BuildRequires: %{python_pfx}-simplejson
-BuildRequires: python-backoff >= 1.9.0
+BuildRequires: python3-backoff >= 1.9.0
```

Comment:
Looking at the lines above, should we use `%{python_pfx}-backoff`?

Reply:
backoff is not available as python-backoff in official repos (it has never been) and python_pfx is set only to `python` (missing the 3 there)

Reply:
but on the other side the package does not exist for el < 8 so I think we can use it. WDYT?